modules/sync/admin-sync.php

```diff
@@ -269,7 +269,7 @@ private function get_data_from_request( array $post_data ): array {
 			return array();
 		}
 
-		if ( $post_data['post_type'] !== $_GET['post_type'] || ! $this->model->is_translated_post_type( $post_data['post_type'] ) ) {
+		if ( $post_data['post_type'] !== $_GET['post_type'] || ! pll_is_translated_post_type( $post_data['post_type'] ) ) {
```

Comment:
Not sure @Chouby will be very fond of it.
I wonder if we should have a dedicated function for this.
Let's see!

Reply:
Are you with me or against me?!
![Woman Screaming At Cat](https://github.com/user-attachments/assets/4308be50-8b6b-4486-97cc-73e4d95f1050)